components/prism-crystal.js

```diff
@@ -0,0 +1,53 @@
+(function(Prism) {
+	Prism.languages.crystal = Prism.languages.extend('ruby', {
+		keyword: [
+			/\b(?:abstract|alias|as|asm|begin|break|case|class|def|do|else|elsif|end|ensure|enum|extend|for|fun|if|ifdef|include|instance_sizeof|lib|macro|module|next|of|out|pointerof|private|protected|rescue|return|require|self|sizeof|struct|super|then|type|typeof|undef|union|unless|until|when|while|with|yield|__DIR__|__FILE__|__LINE__)\b/,
+			{
+        pattern: /(\.\s*)(?:is_a|responds_to)\?/,
```

Comment:
Well, there is no reason. And this is a good idea. I'll definitely think about adding it. ;)